### DISCUSSION TITLE
how to disable Transient prompt

### DISCUSSION THREAD

**Discussion by sixwaaaay** [JanDeDobbeleer/oh-my-posh#1527] [Troubleshoot] [answered]
I enabled the Transient prompt,but  I dont know how to disable it.

**Comment by JanDeDobbeleer** [comment, marked as answer]
Remove the enable statement from your shell's rc or profile. Then restart the shell and it's gone.

**Reply by sixwaaaay** [reply]
thanks a lot

**Reply by ErrorPower2001** [reply]
(sry poor English) In my pwsh's profile, there is only the following code about oh-my-posh:

`oh-my-posh init pwsh --config $env:POSH_THEMES_PATH/clean-detailed.omp.json | Invoke-Expression`

What should I do to disable it?

**Reply by JanDeDobbeleer** [reply]
@sixwaaaay export and edit `clean-detailed.omp.json` to remove the transient prompt OR initialise using  `oh-my-posh init pwsh --config $env:POSH_THEMES_PATH/clean-detailed.omp.json --manual | Invoke-Expression` which also removes tooltips.

**Reply by ErrorPower2001** [reply]
Thanks a lot. every times i have edited `clean-detailed.omp.json` and updated OMP it come back, i will use the second way.

**Reply by JanDeDobbeleer** [reply]
@ErrorPower2001 you can copy that theme to a different location and reference that in the `--config` flag. The `$env:POSH_THEMES_PATH` folder is internal and recreated on update.
